public/js/popup.js

```diff
@@ -26,6 +26,37 @@ function locationToString(location) {
     return `[${start}] - [${end}]`;
 }
 
+function getLineFromFile(code, location) {
+    const [[startLine]] = location;
+    const lines = code.split('\n');
+
+    return lines[startLine - 1];
+}
+
+async function fetchCodeLine(event, name, version, file, location) {
+    const [target] = event.srcElement.getElementsByClassName('tooltip')
+    target.innerText = "Loading ...";
+
+    const response = await fetch(`https://unpkg.com//${name}@${version}/${file}`);
```

Comment:
I guess the whole "file" stream is not required here and the code can be written as follow:

```js
const code = await fetch(`https://unpkg.com//${name}@${version}/${file}`).then((response) => response.text());
```